### PR TITLE
feat(base): add top 20 base coins

### PR DESCRIPTION
**Pull request by jh2oman**
Uses `scripts/add-erc20-tokens-coingecko.ts` to add top 20 base coins. Does not make any of them swapable.

```
yarn ts-node scripts/add-erc20-tokens-coingecko.ts --category-id=base-ecosystem --platform-id=base --viem-chain-id=base --tokens-info-file-path=src/data/mainnet/base-tokens-info.json
```

https://linear.app/valora/issue/ACT-1129/e2e-testing-on-base

### CODE REVIEW

File: src/data/mainnet/base-tokens-info.json [hunk 1]
```diff
@@ -20,5 +20,158 @@
...
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/CTSI.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/cartesi"
+  },
+  {
+    "name": "Curve.Fi USD Stablecoin",
+    "symbol": "crvUSD",
+    "decimals": 18,
+    "address": "0x417ac0e078398c154edfadd9ef675d30be60af93",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/crvUSD.png",
```
Comment (MuckT) [inline src/data/mainnet/base-tokens-info.json:38]: @jh2oman is there an associated asset for this token?

File: assets/.DS_Store
Comment (MuckT) [inline assets/.DS_Store:1]: What's up with this file?

File: src/data/mainnet/base-tokens-info.json (hunk 1 above)
Comment (jh2oman) [reply src/data/mainnet/base-tokens-info.json:38]: The file already exists! same one from another chain

File: assets/.DS_Store
Comment (jh2oman) [reply assets/.DS_Store:1]: I think this is a mac thing that accidentally got added. I'll remove

File: src/data/mainnet/base-tokens-info.json [hunk 2]
```diff
@@ -20,5 +20,158 @@
...
+    "decimals": 18,
+    "address": "0x4ed4e862860bed51a9570b96d89af5e1b0efefed",
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/DEGEN.png",
+    "isNative": false,
+    "infoUrl": "https://www.coingecko.com/en/coins/degen-base"
+  },
+  {
+    "name": "Propy",
+    "symbol": "PRO",
+    "decimals": 18,
```
Comment (MuckT) [inline src/data/mainnet/base-tokens-info.json:108]: Think this token is 8 decimals.

Comment (jh2oman) [reply src/data/mainnet/base-tokens-info.json:108]: Hmm, you're right. Now i feel the need to double check all of them

**Review by MuckT — APPROVED** [review]

**Comment by jh2oman** [comment]
This is actually the top 21-40 coins 🤭

**Pull request merged**